### PR TITLE
Upgrade to netstandard1.6

### PR DESCRIPTION
**Pull request by Jetski5822**
1.6 seems to be the best version we can upgrade too.

Moved from Approvals to Assent so that we can use netstandard.

Please review @wis3guy

Edit: All tests now green.

### CODE REVIEW

File: WebApi.Hal/Hypermedia.cs [hunk 1]
```diff
@@ -8,23 +8,24 @@ namespace WebApi.Hal
 {
     public class Hypermedia : IHypermediaResolver, IHypermediaBuilder
     {
-        readonly IDictionary<Type, Link> selfLinks = new Dictionary<Type, Link>();
-        readonly IDictionary<Type, IList<Link>> hypermedia = new Dictionary<Type, IList<Link>>();
-        readonly IDictionary<Type, object> appenders = new Dictionary<Type, object>();
+        private readonly IDictionary<Type, Link> selfLinks = new Dictionary<Type, Link>();
```
[the remark this answers is not in the document]
Comment (Jetski5822) [reply WebApi.Hal/Hypermedia.cs]: What would the visibility be if not explicit?is it protected in this instance? I thought it was Private...

File: WebApi.Hal/JsonHalMediaTypeInputFormatter.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
+using System;
```
[the remark this answers is not in the document]
Comment (Jetski5822) [reply WebApi.Hal/JsonHalMediaTypeInputFormatter.cs]: yup

File: WebApi.Hal/Representation.cs [hunk 3]
```diff
@@ -12,28 +11,27 @@ namespace WebApi.Hal
 {
     public abstract class Representation : IResource
     {
-        protected Representation()
```
[the remark this answers is not in the document]
Comment (Jetski5822) [reply WebApi.Hal/Representation.cs]: I removed it in favour of property initialisation;

File: Build.cmd [hunk 4]
```diff
@@ -1,5 +0,0 @@
-@echo off
```
[the remark this answers is not in the document]
Comment (Jetski5822) [reply Build.cmd]: dotnet pack ?

File: WebApi.Hal/Hypermedia.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (panmanphil) [reply WebApi.Hal/Hypermedia.cs]: you are correct, this should be fine

File: WebApi.Hal.Tests/WebApi.Hal.Tests.csproj [hunk 5]
```diff
@@ -1,110 +1,31 @@
...
-    <Compile Include="Representations\OrganisationWithPeopleDetailRepresentation.cs" />
-    <Compile Include="Representations\OrganisationWithPeopleRepresentation.cs" />
-    <Compile Include="Representations\ProductRepresentation.cs" />
-    <Compile Include="ResolvingHalResourceTest.cs" />
-    <Compile Include="UriBuilderTests.cs" />
+    <PackageReference Include="Assent" Version="1.0.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="1.1.1" />
+    <PackageReference Include="xunit" Version="2.3.0-beta3-build3705" />
```
[the remark this answers is not in the document]
Comment (Jetski5822) [reply WebApi.Hal.Tests/WebApi.Hal.Tests.csproj]: Fixed

File: WebApi.Hal/WebApi.Hal.csproj [hunk 6]
```diff
@@ -1,101 +1,18 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```
[the remark this answers is not in the document]
Comment (Jetski5822) [reply WebApi.Hal/WebApi.Hal.csproj]: Updated to netstandard2.0

**Comment by Jetski5822** [comment]
Is there anything you want me to change to get this merged in?

**Comment by panmanphil** [comment]
Thanks again for all the work that went into this pull request. We can start a 3.0 release track now with MVC Core 2 and netstandard2 bringing this api into the modern world

**Pull request merged**